### PR TITLE
fix: adapter features messaging

### PR DESCRIPTION
**Pull request by ematipico**
## Changes

The phrasing was misleading. This PR fixes it

## Testing

CI should pass

## Docs

 /cc @withastro/maintainers-docs for feedback!

### CODE REVIEW

File: packages/astro/src/integrations/features-validation.ts [hunk 1]
```diff
@@ -124,20 +124,20 @@ function validateSupportKind(
 }
 
 function featureIsUnsupported(adapterName: string, logger: Logger, featureName: string) {
-	logger.error('config', `The feature "${featureName}" is not supported (used by ${adapterName}).`);
+	logger.error('config', `The adapter ${adapterName} doesn't support the feature "${featureName}" yet.`);
```
Comment (sarah11918) [inline packages/astro/src/integrations/features-validation.ts:127]:
```suggestion
	logger.error('config', `The adapter ${adapterName} doesn't currently support the feature "${featureName}".`);
```
This implies that it *will* support the feature, and we might not want to make that promise. I think it's fine just to say that it doesn't currently support it!

File: packages/astro/src/integrations/features-validation.ts [hunk 2]
```diff
@@ -124,20 +124,20 @@ function validateSupportKind(
...
-		`The feature "${featureName}" is experimental and subject to change (used by ${adapterName}).`,
+		`The adapter ${adapterName} set experimental support for "${featureName}", which means that things might not work, or could change.`,
 	);
 }
 
 function featureIsDeprecated(adapterName: string, logger: Logger, featureName: string) {
 	logger.warn(
 		'config',
-		`The feature "${featureName}" is deprecated and will be removed in the future (used by ${adapterName}).`,
+		`The adapter ${adapterName} has deprecated its support for "${featureName}", which means that the feature might not work anymore in the future for this adapter.`,
```
Comment (sarah11918) [inline packages/astro/src/integrations/features-validation.ts:140]:
```suggestion
		`The adapter ${adapterName} has deprecated its support for "${featureName}", and future compatibility is not guaranteed. The adapter may completely remove support for this feature without warning.`,
```

File: packages/astro/src/integrations/features-validation.ts [hunk 3]
```diff
@@ -124,20 +124,20 @@ function validateSupportKind(
...
 function featureIsUnsupported(adapterName: string, logger: Logger, featureName: string) {
-	logger.error('config', `The feature "${featureName}" is not supported (used by ${adapterName}).`);
+	logger.error('config', `The adapter ${adapterName} doesn't support the feature "${featureName}" yet.`);
 }
 
 function featureIsExperimental(adapterName: string, logger: Logger, featureName: string) {
 	logger.warn(
 		'config',
-		`The feature "${featureName}" is experimental and subject to change (used by ${adapterName}).`,
+		`The adapter ${adapterName} set experimental support for "${featureName}", which means that things might not work, or could change.`,
```
Comment (sarah11918) [inline packages/astro/src/integrations/features-validation.ts:133]:
```suggestion
		`The adapter ${adapterName} provides experimental support for "${featureName}". You may experience issues or breaking changes until this feature is fully supported by the adapter.`,
```

File: .changeset/long-months-burn.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+---
+'astro': patch
+---
+
+Changes the phrasing of the adapter features because they were misleading.
```
Comment (sarah11918) [inline .changeset/long-months-burn.md:5]:
```suggestion
Changes messages logged when using unsupported, deprecated or experimental adapter features for clarity
```

**Review by sarah11918 — COMMENTED** [review]
Thanks for tackling this so quickly @ematipico ! Here are some thoughts from me. See what you think!

File: packages/astro/src/integrations/features-validation.ts (hunk 1 above)
Comment (ematipico) [reply packages/astro/src/integrations/features-validation.ts:127]: Good call!

**Review by matthewp — APPROVED** [review]

File: .changeset/long-months-burn.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+---
+'astro': patch
+---
+
+Changes messages logged when using unsupported, deprecated or experimental adapter features for clarity
```
Comment (yanthomasdev) [inline .changeset/long-months-burn.md:5]: Oxford comma nit
```suggestion
Changes messages logged when using unsupported, deprecated, or experimental adapter features for clarity
```

**Review by florian-lefebvre — APPROVED** [review]

**Pull request merged**